db/migrate/20180511210539_noramlize_druid_name.rb

```diff
@@ -0,0 +1,5 @@
+class NoramlizeDruidName < ActiveRecord::Migration[5.1]
+  def change
+    PreservedObject.normalize_druid_name
+  end
+end
```

Comment:
I like the approach. I hadn't thought much about using migrations to remediate data, as opposed to changing a schema.